harmony/__init__.py

```diff
@@ -7,7 +7,7 @@
 """
 
 # Automatically updated by `make build`
-__version__ = "v1.0.26"
+__version__ = "1756beta"
```

Comment:
nit: shouldn't have checked in this change. The official release version here looks better than your test release.

Reply:
Hmmm - I don't recall doing that - mabye that got checked in by the build process?

Reply:
Fixed